.github/workflows/replace-expo-qr-codes.yml

```diff
@@ -0,0 +1,96 @@
+name: Replace Expo update ID in README
+
+on:
+  workflow_call:
+
+jobs:
+  replace-expo-update-id:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install EAS cli
+        uses: expo/expo-github-action@v8
+        with:
+          expo-cache: true
+          expo-version: latest
+          eas-version: latest
+          token: ${{ secrets.EXPO_TOKEN }}
+
+      - name: Get Expo update ID
+        id: get-expo-update-id
+        run: |
+          UPDATES=$(eas update:view --json $(eas branch:view production --json --non-interactive | jq ".currentPage[0].group" -r ))
+          echo "::set-output name=android_update_id::$(echo $UPDATES | jq -r '.[] | select(.platform | contains("android")) | .id')"
+          echo "::set-output name=ios_update_id::$(echo $UPDATES | jq -r '.[] | select(.platform | contains("ios")) | .id')"
+
+      - name: Replace Expo update ID
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.PAT }}
```

Comment:
ok no we can't do this. let's move this to the website repo and run on a schedule instead

Reply:
What do you think would be a good frequency for running it? since it will just exit if there's nothing to update I think it could be run relatively frequently.

Reply:
once per hour?